stubs/pywin32/win32com/server/factory.pyi

```diff
@@ -0,0 +1,9 @@
+from collections.abc import Iterable
+from typing import Any
+
+from _win32typing import PyIClassFactory, PyIID
+
+def RegisterClassFactories(
+    clsids: Iterable[PyIID], flags: int | None = None, clsctx: int | None = None
+) -> list[tuple[PyIClassFactory, int]]: ...
+def RevokeClassFactories(infos: Iterable[tuple[Any, int]]) -> None: ...  # The first element is not used in the original code.
```

Comment:
Not asking for a change, just FYI, we do have `_typeshed.Unused` for when it doesn't cause variance issues (like a param, or a `Callable` return type)